tutorials/basic-topics/linear-search.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Linear search'
+description: 'Linear search is a searching technique that works in a linear/sequential fashion. It traverses whole of the data structure and returns the position(s) where the value is found. If not found then it simple returns nothing.'
+hide_table_of_contents: true
+---
+
+<TutorialAuthors names="@siddoinghisjob"/>
+
+## Overview
+
+Lets say we have a linear data structure - array, linked list - and we need to search for a certain element. To search there are mainly two searching techniques, one being linear search. In linear search we traverse the whole array and then while traversing we check for the particular item. If there's a match then we return that position( we can return multiple positions too).
+
+<Tabs>
+<TabItem value="cpp" label="C++">
+<SolutionAuthor name="@siddoinghisjob"/>
+
+```cpp
+#include<iostream>
+using namespace std;
+
+int main(){
+  // Array to keep the values
+  int arr[5] = {1,2,3,4,5};
+  // Item to be searched 
+  int item = 3; 
+  for(int i = 0; i < 5; i++){
+    // checking if item is arr[i]
+    if(arr[i] == item){  
+      cout << "Found at " << i << endl;
+    }
+  }
+} 
+```
+
+</TabItem>
+</Tabs>
+
+## Example problems
```

Comment:
The format is not desired. Please check out other tutorials. 
e.g. https://github.com/wingkwong/leetcode-the-hard-way/blob/main/tutorials/basic-topics/prefix-sum.md?plain=1